kubernetes/helm/templates/client-config.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.client.enabled }}       
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "name-client" . }}-nginx
+data:
+  nginx.conf: |
+    worker_processes  auto;
+
+    error_log  /home/nginx/log/nginx/error.log notice;
+    pid        /home/nginx/nginx.pid;
+
+    events {
+        worker_connections  1024;
+    }
+
+    stream {
+      include /etc/nginx/conf.d/*.conf;
+    }
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "name-client" . }}-env
+stringData:
+  INVITE_TOKEN: {{ quote .Values.peering.psk }}
```

Comment:
that process has many names, invite, peering, syncing, helloing :D

Reply:
True, I'll add an issue for that